packages/mdc-fab/_mixins.scss

```diff
@@ -16,7 +16,22 @@
 @import "@material/animation/variables";
 @import "@material/elevation/mixins";
 @import "@material/ripple/mixins";
+@import "@material/theme/functions";
 @import "@material/theme/mixins";
+@import "./variables";
+
+@mixin mdc-fab-accessible($container-color) {
+  @include mdc-fab-container-color($container-color);
+  $light-or-dark: mdc-theme-light-or-dark($container-color);
+  @if ($light-or-dark == 'light') {
```

Comment:
It might be helpful to have a comment here about how `'light'` refers to the text color rather than the background color... I had to look it up to convince myself this code wasn't backwards.